docs/conf/portland/2025/news/announcing-speakers.rst

```diff
@@ -0,0 +1,56 @@
+:template: {{year}}/generic.html
+:og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
+
+.. post:: Feb 7, 2025
+   :tags: {{shortcode}}-{{year}}, speakers, tickets, visiting, sponsors
+
+Announcing Portland conference speakers
+=======================================
+
+Our Portland conference is almost 3 months away, and we're super excited to see you all there.
+Now is the perfect time to buy your tickets and arrange your travel.
+
+Today's news is the list of awesome speakers and their talks.
+A huge thank you to everyone who submitted a proposal;
+our event wouldn't be possible without you!
+
+{{year}} {{city}} conference speakers
+----------------------------------------
```

Comment:
FWW, I've been using markdown for posts, and it makes this way less awkward than trying to guess the length :)

Reply:
Yeah, this is one of the *super* annoying things about Sphinx.